docs/pywbemcligeneraloptions.rst

```diff
@@ -74,6 +74,13 @@ interactive mode and as a table:
 command, general the options redefinition's are not retained between command
 executions.
 
+The general options may be used in the interactive mode to override
+the default general options or specific general options for the current command.
+
+Thus, behavior of pywbemcli (ex. :ref:`--output-format`) or the parameters of the
```

Comment:
Change invalid reference ":ref:`--output-format`" to ":ref:`--output-format general option`".

Reply:
DONE